proto/addressmetadata.proto

```diff
@@ -1,39 +1,74 @@
 syntax = "proto3";
 package models;
 
-message Header {
-    string Name = 1;
-    string Value = 2;
+// Specifies the revocation method
+enum RevocationMethod {
+    /* If any of the pubkeys given in commitment_data[..n-1] signs
+    data commitment_data[n] then revocation is granted. */
+    BackupPubkey = 0;
```

Comment:
This is used as a place holder and the most simple revocation method. Enum's can't be empty.

Reply:
The zero element should be "Not revoked."

The 1st enum element should simply be "Revoked." 

the third can be the following scheme (Although I think we should omit this for the time being)

You have two secret keys:
`a`, and `b`
pubkeys: `aG` and `bG`
and the current pubkey being examined is:
```
P = bG + hash(aG||bG)G
```
To revoke `P` you reveal aG and bG.  Then you sign a redirection to some new public key `cG`

So the revocation message needs to have: aG, bG, cG, and a signature of cG by a.   This method requires that you *generate* your public address using a secret offline key `a`.

Reply:
> The zero element should be "Not revoked."
> The 1st enum element should simply be "Revoked."

This is supposed to the the revocation method, established apriori. Rather than the state of revocation e.g. revoked vs not revoked.